src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -563,7 +563,7 @@ pal::string_t fx_muxer_t::resolve_cli_version(const pal::string_t& global_json)
     return retval;
 }
 
-pal::string_t resolve_sdk_version(pal::string_t sdk_path)
+pal::string_t resolve_sdk_version(pal::string_t sdk_path, bool parse_only_production)
```

Comment:
"parse_only_production" is being passed into 2 methods; it must be the same value to get consistent behavior...
lines:720-722
            bool parse_only_production = false;  // false -- implies both production and prerelease.
            pal::string_t new_cli_version = resolve_sdk_version(current_sdk_path, parse_only_production);
            if (higher_sdk_version(new_cli_version, &cli_version, parse_only_production))